frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.init

```diff
@@ -95,6 +95,9 @@ start_orchestrator() {
   if [[ -n "${orchestrator_cvd_artifacts_dir}" ]]; then
     args+=("--cvd_artifacts_dir=${orchestrator_cvd_artifacts_dir}")
   fi
+  if [[ -n "${orchestrator_cvd_creation_timeout}" ]]; then
+    args+=("--cvd_creation_timeout=${orchestrator_cvd_creation_timeout}")
```

Comment:
+1 with @ser-io's comment to add unit suffix like `cvd_creation_timeout_secs`.

Ditto here.

Reply:
Added unit for the variable, too.